### PR TITLE
Make tests work on Node

### LINKED ISSUE #4: Make test's work in NodeJS
We'll probably have to re-write tests in Mocha or Jasmine.

We'll need to import can-simple-dom and make sure all the DOM tests pass.

### PR DESCRIPTION
**Pull request by andrejewski**
Closes #4

### CODE REVIEW

**Comment by andrejewski** [comment]
The tests are very flaky due to `mocha-qunit-ui`'s handling of asynchronous tests. Even if this passes CI, I'd like a better solution that is less of a randomized headache.

**Comment by andrejewski** [comment]
Using QUnit 2 instead of `mocha-qunit-ui` for the DOM tests is much more reliable.

File: package.json [hunk 1]
```diff
@@ -34,18 +35,21 @@
   "devDependencies": {
     "async": "^2.2.0",
     "bit-docs": "0.0.7",
-    "can-vdom": "^3.1.0-pre.1",
+    "can-simple-dom": "^1.1.1",
```
Comment (phillipskevin) [inline package.json]: is `can-simple-dom` needed?

Comment (phillipskevin) [reply package.json]: since it is a dependency of `can-vdom` and it doesn't look like it's used directly here.

Comment (andrejewski) [reply package.json]: It's not needed, good catch

**Review by phillipskevin — APPROVED** [review]
looks good

**Pull request merged**